src/pymordemos/output_error_estimation.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+# This file is part of the pyMOR project (https://www.pymor.org).
+# Copyright 2013-2021 pyMOR developers and contributors. All rights reserved.
+# License: BSD 2-Clause License (https://opensource.org/licenses/BSD-2-Clause)
+
+
+import numpy as np
+from typer import Argument, run
+
+from pymor.basic import *
+
+
+def main(
+    grid_intervals: int = Argument(..., help='Grid interval count.'),
+    training_samples: int = Argument(..., help='Number of samples used for training the reduced basis.'),
+    verification_samples: int = Argument(..., help='Number of samples used for verification of the output error.')
+):
+    set_log_levels({'pymor': 'WARN'})
+    """Example script for using the DWR output error estimation"""
```

Comment:
This is not yet true in this pr, or?
```suggestion
    """Example script for using output error estimation"""
```